os/drivers/serial/Kconfig

```diff
@@ -11,8 +11,9 @@ config DEV_LOWCONSOLE
 	---help---
 		Use the simple, low-level, write-only serial console driver (minimal support)
 
-config SERIAL_REMOVABLE
```

Comment:
You should not remove this.

Reply:
At the previous time, I remember receiving the request that I should change it..
Please check the final direction..

Reply:
I will fix it

Reply:
This is a part of usb. Both should be in this Kconfig.
Please revert this removing.

Reply:
I have already reverted it